tests/pdk/finfet_pdk/test_circuits.py

```diff
@@ -19,7 +19,8 @@ def test_cmp_vanilla():
     netlist = circuits.comparator(name)
     constraints = [
         {"constraint": "PowerPorts", "ports": ["vccx"]},
-        {"constraint": "GroundPorts", "ports": ["vssx"]}
+        {"constraint": "GroundPorts", "ports": ["vssx"]},
+        {"constraint": "ConfigureCompiler", "same_template": False}
```

Comment:
Why is this necessary?

Reply:
It was a test as I was getting one DRC while running this on my system and merged wrongly. 
This was not necessary and will get rid of it.